### PR TITLE
Move location to main screen and add pricing tier label

### PR DESCRIPTION
**Pull request by LazarusX**
image

### CODE REVIEW

File: IoT/Files/Parameters/IoTHubParameters.json [hunk 1]
```diff
@@ -74,7 +74,8 @@
             ],
             "defaultValue": "S1",
             "properties": {
-                "attributes": "summarizable;"
+                "attributes": "summarizable;",
+                "summaryText": "{{{inputs.iotHubSku}}}"
```
Comment (bishal-pdMSFT) [inline IoT/Files/Parameters/IoTHubParameters.json:78]: This change should not be required. If you do not specify any summaryText the input name is picked by default. SummaryText can be be used to specify some custom text e.g. dependent on some other input.

**Review by bishal-pdMSFT — APPROVED** [review]
Please see the above comment.

File: IoT/Files/Parameters/IoTHubParameters.json (hunk 1 above)
Comment (LazarusX) [reply IoT/Files/Parameters/IoTHubParameters.json:78]: Thanks for the feedback!

However, it looks like after removing the `summaryText` line, IoT Hub SKU no longer shows in the main screen:

image

Here is how it looks like before the change:

image

Comment (bishal-pdMSFT) [reply IoT/Files/Parameters/IoTHubParameters.json:78]: yes, it seems I was wrong about that. You are good to go.

**Comment by LazarusX** [comment]
Thanks @bishal-pdMSFT for the review!

When is your next release?

**Comment by bishal-pdMSFT** [comment]
We can rollout next deployment by next week.

**Comment by LazarusX** [comment]
Thanks @bishal-pdMSFT!

Please help merge this PR.

**Pull request merged**